### ISSUE TITLE
Support for OAuth (vs. API Token)?

### ISSUE THREAD

**Issue by fvescia** [ropensci/qualtRics#318]
Hello! [The Qualtrics documentation recommends using OAuth instead of an API token to authorize API calls.](https://api.qualtrics.com/b626f29209735-things-to-know-about-the-api#using-the-api-token-is-less-secure-than-oauth) 

Does qualtRics support OAuth? Based on [issue 209](https://github.com/ropensci/qualtRics/issues/209) it seems like not, but I wanted to confirm. Thanks!

**Comment by juliasilge** [comment]
That's right; we don't currently support OAuth credentials in this R package. The way the Qualtrics API handles OAuth credentials has changed a lot over the last few years and seems much more usable now, so it may be worth considering adding support for those credentials. 👍

**Comment by fvescia** [comment]
Got it, thanks for confirming!

**Comment by abourdea** [comment]
Hi, I'm interested in whether there has been any movement on adding support for OAuth/whether this is in qualtRics's future?

**Comment by juliasilge** [comment]
I think it would be a good thing for us to add, yep! I don't use Qualtrics anymore personally (see https://github.com/ropensci/qualtRics/issues/383) so it would be great to have someone contribute this feature who has more hands-on experience with the API. I also wonder if we should switch the package over to use httr2 so we can get this nicer handling: <https://httr2.r-lib.org/articles/oauth.html>
